### PR TITLE
Update in data connector and parser

### PR DESCRIPTION
**Pull request by v-rbajaj**
Required items, please complete
   
   Change(s):
   - See guidance below

   Reason for Change(s):
   - See guidance below

   Version Updated:
   - Required only for Detections/Analytic Rule templates
   - See guidance below

   Testing Completed:
   - [Test Document](https://microsoftapc.sharepoint.com/:w:/r/teams/EcosystemCATTeam/_layouts/15/Doc.aspx?sourcedoc=%7B7DE564F4-11EF-4284-88FC-41E7126BBCBF%7D&file=ISC%20Bind%20Repackaging.docx&action=default&mobileredirect=true&share=IQH0ZOV97xGEQoj8QecSa7y_AU1KGEzhHi-HSkSl7qyuSko)

   Checked that the validations are passing and have addressed any issues that are present:
   - See guidance below

# Guidance <- remove section before submitting
-----------------------------------------------------------------------------------------------------------
## Before submitting this PR please ensure that you have read the following sections and filled out the changes, reason for change and testing complete sections:

Thank you for your contribution to the Microsoft Sentinel Github repo.

> Details of the code changes in your submitted PR.  Providing descriptions for pull requests ensures there is context to changes being made and greatly enhances the code review process.  Providing associated Issues that this resolves also easily connects the reason.
   
   Change(s):
   - Updated syntax for XYZ.yaml

   Reason for Change(s):
   - New schema used for XYZ.yaml
   - Resolves ISSUE #1234

   Version updated:
   - Yes
   - Detections/Analytic Rule templates are required to have the version updated

> The code should have been tested in a Microsoft Sentinel environment that does not have any custom parsers, functions or tables, so that you validate no incorrect syntax and execution functions properly.  If your submission requires a custom parser or function, it must be submitted with the PR.

   Testing Completed:
   - Yes/No/Need Help

_Note: If updating a detection, you must update the version field._

> Before the submission has been made, please look at running the KQL and Yaml Validation Checks locally.
> https://github.com/Azure/Azure-Sentinel#run-kql-validation-locally

   Checked that the validations are passing and have addressed any issues that are present:
   - Yes/No/Need Help
   
   _Note: Let us know if you have tried fixing the validation error and need help._

> References: 
> - [Guidance for Detection checks](https://github.com/Azure/Azure-Sentinel#pull-request-detection-template-structure-validation-check)
> - [General contribution guidance](https://github.com/Azure/Azure-Sentinel/wiki#what-can-you-contribute-and-how-can-you-create-contributions)
> - [PR validation troubleshooting](https://github.com/Azure/Azure-Sentinel#pull-request)

-----------------------------------------------------------------------------------------------------------

### CODE REVIEW

**Comment by v-atulyadav** [comment]
Looking into this

File: Solutions/ISC Bind/Data Connectors/Connector_Syslog_ISCBind.json [hunk 1]
```diff
@@ -62,7 +62,7 @@
   "instructionSteps": [
     {
       "title": "",
-      "description": ">This data connector depends on a parser based on a Kusto Function to work as expected. [Follow the steps](https://aka.ms/sentinel-iscbind-parser) to use the Kusto function alias, **ISCBind**",
+      "description":"**NOTE:** This data connector depends on a parser based on a Kusto Function to work as expected which is deployed as part of the solution. To view the function code in Log Analytics, open Log Analytics/Microsoft Sentinel Logs blade, click Functions and search for the alias ISCBind and load the function code or click [here](https://github.com/Azure/Azure-Sentinel/blob/master/Solutions/ISC%20Bind/Parsers/ISCBind.txt).The function usually takes 10-15 minutes to activate after solution installation/update." ,
```
Comment (v-prasadboke) [inline Solutions/ISC Bind/Data Connectors/Connector_Syslog_ISCBind.json:65]: Please change this text to  [ **NOTE:** This data connector depends on a parser based on a Kusto Function to work as expected which is deployed as part of the solution. To view the function code in Log Analytics, open Log Analytics/Microsoft Sentinel Logs blade, click Functions and search for the alias IscBind and load the function code or click [here](https://github.com/Azure/Azure-Sentinel/blob/master/Solutions/ISC%20Bind/Parsers/ISCBind.txt), on the second line of the query, enter the hostname(s) of your IscBind device(s) and any other unique identifiers for the logstream. The function usually takes 10-15 minutes to activate after solution installation/update.

Comment (v-rbajaj) [reply Solutions/ISC Bind/Data Connectors/Connector_Syslog_ISCBind.json:65]: Hi Prasad, The current description in the PR is in lined to Parser's description so we don't need to change it .

**Review by v-vdixit — APPROVED** [review]
Looks good!

**Pull request merged**